module/applications/actor/rest/short-rest-dialog.mjs

```diff
@@ -79,6 +80,18 @@ export default class ShortRestDialog extends BaseRestDialog {
       value: context.config.autoHD
     });
 
+    const numericalDenom = Number(context.hitDice.denomination?.slice(1));
+    if ( numericalDenom ) {
+      const { value: currHP, max: maxHP } = this.actor.system.attributes.hp;
+      context.healthBarText = `${game.i18n.localize("DND5E.HP")}: ${currHP} / ${maxHP}`;
+      context.progressBar = 100 * currHP / maxHP;
+      let potentialRegain = numericalDenom;
```

Comment:
Should make sure that we are taking the characters con modifier into account here. Perhaps we can display the bar in such a way so that the minimum regained hit points (con mod + 1, min of 1) is shown in red, and the potential range based on the die roll is shown in a more transparent color.